.env

```diff
@@ -22,4 +22,12 @@ REACT_APP_SWR_LOG_CACHE=false
 REACT_APP_SWR_LOG_REQUESTS=false
 
 # This is the title displayed while the UI is initially loading in the browser
-REACT_APP_DEFAULT_PAGE_TITLE=Estuary Flow
\ No newline at end of file
+REACT_APP_DEFAULT_PAGE_TITLE=Estuary Flow
+
+# LogRocket settings
+REACT_APP_LOGROCKET_APP_ID=tue20y/estuarytest
+REACT_APP_LOGROCKET_SANITIZE_INPUTS=true
+REACT_APP_LOGROCKET_SANITIZE_TEXT=false
+REACT_APP_LOGROCKET_ID_USER=true
+REACT_APP_LOGROCKET_ID_USER_INCLUDE_NAME=true
+REACT_APP_LOGROCKET_ID_USER_INCLUDE_EMAIL=true
```

Comment:
Trying to make configuring this as granular as possible. Important for anyone running their own version of the UI.